migration/main.py

```diff
@@ -13,16 +14,43 @@
 from manager import AccountManagerFactory, CourseManager
 from utils import convert_csv_to_int_list, find_entity_by_id, time_execution
 
+summaryLogBuffer = StringIO()
+summaryLogHandler = logging.StreamHandler(summaryLogBuffer)
+summaryLogHandler.setLevel(logging.WARNING)
```

Comment:
Would we ever need to make this configurable?

Reply:
Probably not.  The app has two other log level params already and it might get confusing to have yet another.  The idea behind this one is just to collect all log messages that are warning or higher.